### ISSUE TITLE
ISA published via bulk publishing : assets are publicly available, BUT not shared with own project

### ISSUE THREAD

**Issue by OlgaKrebs** [seek4science/seek#1882]
I had created ISA with assets and kept it privat (+shared with one person only) and later on - as paper was published I shared it with public using "bulk sharing" and it worked! but later I realised that whole ISA is NOT shared with own project it belongs to. It looks like bulk sharing procedure ONLY change sharing for "public" and does not touch other project listed  on "manage"  page

**Comment by OlgaKrebs** [comment]
![publish_balk_problem](https://github.com/seek4science/seek/assets/55085007/be9e7b91-d841-4e1a-a2bd-e302bd289ccc)

**Comment by OlgaKrebs** [comment]
This happened on LiSyM SEEK, I am not sure how it is on FAIRDOMHub

**Comment by OlgaKrebs** [comment]
during the bulk publishing thete is no option for user to say "share with my project"

**Comment by stuzart** [comment]
no, during bulk publishing it should set the overall access to public, but leave the project permissions untouched. If you want to change the project permissions then you use batch permission changes

**Comment by stuzart** [comment]
I've just tested this on our test server, both publishing a single Assay and also via Publish My Items button, and the permissions for the projects were left unaffected, so I think this has been fixed.
There have been some changes to this code since 1.14.2, combining Batch Permission Changes and Batch Publishing, so quite a bit of re-rewriting and testing

**Comment by stuzart** [comment]
Are you able to test it again on Lisym SEEK, say creating a temporary Assay, sharing with a couple of projects, and then publish and go back to look at the sharing permissions and check the projects? And then try exactly the same on https://testing.sysmo-db.org or even FAIRDOMHub if you can delete it afterwards.

**Comment by stuzart** [comment]
As commented above, the publishing just changes the permissions for public, and doesn't affect other permissions. Would need to be careful about affecting the other permissions, but as Olga found, it can be a bit odd to be left in a position where the item is visible publically but not whilst logged in. Perhaps it should give a warning and option to make sure other permissions aren't set to be less permissive.
